ext/storage_sources/local_store/local_store.c

```diff
@@ -729,58 +736,58 @@ local_open(WT_FILE_SYSTEM *file_system, WT_SESSION *session, const char *name,
     LOCAL_FILE_SYSTEM *local_fs;
     LOCAL_FLUSH_ITEM *flush;
     LOCAL_STORAGE *local;
-    WT_FILE_HANDLE *file_handle;
-    int fd, oflags, ret;
-    char *open_name;
+    WT_FILE_HANDLE *file_handle, *wt_fh;
+    WT_FILE_SYSTEM *wt_fs;
+    struct stat sb;
+    int ret;
+    bool exists;
 
     (void)flags; /* Unused */
 
-    fd = oflags = ret = 0;
+    ret = 0;
     *file_handlep = NULL;
     local_fh = NULL;
     local_fs = (LOCAL_FILE_SYSTEM *)file_system;
     local = local_fs->local_storage;
+    wt_fs = local_fs->wt_fs;
 
-    if (file_type != WT_FS_OPEN_FILE_TYPE_DATA)
-        return (
-          local_err(local, session, EINVAL, "%s: open: only data file types supported", name));
-
-    local->op_count++;
-    if (flags == WT_FS_OPEN_CREATE)
-        oflags = O_WRONLY | O_CREAT;
-    else if (flags == WT_FS_OPEN_READONLY)
-        oflags = O_RDONLY;
-    else
-        return (local_err(local, session, EINVAL, "open: invalid flags: 0x%x", flags));
+    /*
+     * If needed we could relax this, and allow opens of other kinds of files to use the regular
```

Comment:
I suggest rewording this comment because "If needed we could relax this" is the first thing and I have no idea what "this" is nor what constraints you're talking about. Maybe something like:
Restrict opens to regular data files. If needed, ...

also does that mean that we could not open/flush WT log files?

Reply:
We normally wouldn't be using this file system to flush WT log files.  The block manager PR has code like:
```
  WT_WITH_FILE_SYSTEM(....)
```
that is used narrowly, like when creating a data file, or opening a data file or the turtle file.  It would be unexpected to try to open a log file under that umbrella, so I made it an error here so we understand it.  That said, that constraint could be relaxed pretty easily if we wanted.  I'll update the comment to give this context.